.gitignore

```diff
@@ -11,3 +11,4 @@ nbactions.xml
 out
 build
 src/main/java/org/traccar/protobuf
+debug.xml
```

Comment:
Why is this added to ignore file?

Reply:
Shouldn't have pushed, that's just for my local repo.